documentation/home.md

```diff
@@ -0,0 +1,32 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector enables developers to orchestrate agentic experiences from within Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.
+
+## Key Features
+
+The MuleSoft AI Chain Connector simplifies AI integration into MuleSoft applications with:
+
+- Seamless Interaction with LLMs: Effortlessly integrate and utilize LLMs for tasks such as natural language processing, text generation, text classification and more complex language tasks.
+- Embeddings and Search: Leverage embeddings to handle tasks like text similarity, document search, and clustering.
```

Comment:
Can we update the beginning of this bullet to read: "Generate and leverage embeddings"?